packages/web-components/src/components/va-minimal-header/va-crisis-line-modal.tsx

```diff
@@ -0,0 +1,112 @@
+import { Component, Host, State, h } from '@stencil/core';
+import arrowRightSvg from '../../assets/arrow-right-white.svg';
+import { CONTACTS } from '../../../../react-components/src/components/Telephone/contacts';
```

Comment:
This is the React import for CONTACTS (being deprecated). You should use this import instead:

```
import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
```

Reply:
Thanks! I was hoping there was somewhere better to grab this from. Updated import.

Reply:
Hmm, that import does not work in the component library for me...

Reply:
I changed this to pull from /web-components/index.ts which exports CONTACTS and it appears to work.